packages/mobile/src/navigation/account/account.navigation.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import { AccountScreenName } from '~/common/enums/enums';
+import {
+  PersonalInfoScreen,
+  SettingsScreen,
+  SupportScreen,
+  Account,
+  MessagesScreen,
+} from '~/screens/screens';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationOptions,
+} from '@react-navigation/native-stack';
+import { AccountNavigationParamList } from '~/common/types/types';
+import { useCustomTheme, useTranslation } from '~/hooks/hooks';
+import { HeaderLeft } from '~/components/components';
+
+const Stack = createNativeStackNavigator<AccountNavigationParamList>();
+
+const AccountNavigation = () => {
+  const { t } = useTranslation();
+  const { dark, colors } = useCustomTheme();
+
+  const screenOptions: NativeStackNavigationOptions = {
```

Comment:
We don't really need additional navigator for every section. You may use Navigator.Group if you want to separate screen groups in some way